src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,86 @@ redirects:
 </Callout>
 
-## Define policies for entities or people [#entities-people]
+## Recommended alerts [#recommend-alerts]
+
+Use [recommended practices](https://discuss.newrelic.com/t/announcing-alert-recommendations-for-apm-applications/154542) if you are new to Alerts or if you want suggestions that optimize your alert coverage.
+
+## Policies [#policy-practices]
+
+A policy is a container for alike conditions.
+
+<Callout variant="tip">
+Learn how to [create, edit, or find policies](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) if you’re new to Alerts.
+</Callout>
 
-When designing your policies, consider:
+Organize your policy by surrounding its conditions to a single entity and to the specific team that needs to be notified when a problem arises within that entity.
 
-* The parts of your architecture that need personnel to be responsible for them
-* The individuals who are responsible for one or more parts of your infrastructure
+If a team is monitoring several groups of the same entity type, combine those entity clusters (like servers) together into one policy. This way, your team can be notified from one policy rather than navigating several policies at once.
 
-An organization may have multiple entities monitored by APM, browser, infrastructure, and synthetic monitoring. Examples of considerations for different teams:
+Consider your team’s role when assigning them to policies:
 
 * **Software developers** may need [notifications](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-controlling-where-send-alerts) for both front-end and back-end performance, such as webpage response time and page load JavaScript errors.
 * **Operations personnel** may need notifications for poor back-end performance, such as server memory and load averages.
 * **The product owner** may need notifications for positive front-end performance, such as improved end user Apdex scores or sales being monitored in [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-new-relic-one-dashboards).
 
-By following these best practices, key personnel will receive actionable notifications for the metrics that matter to them, and overall, the organization will be able to identify and respond to trends or patterns more efficiently.
+Refrain from creating policies that include multiple teams to keep policies centralized.
+
+## Notification channels [#channel-practices]
+
+Tailor notifications to the most useful channel and policy so you can avoid alert fatigue and help the right personnel receive and respond to incidents they care about in a systematic way.
+
+<Callout variant="tip">
+Learn how to [set up notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/) if you’re new to Alerts.
+</Callout>
+
+Notify teams and individuals who:
```

Comment:
This might be my own personal aesthetic, but I always try to avoid bullets when I only have two of 'em.

You've got "when an incident arises" twice, which makes me think you could collapse this into a single sentence pretty easily.

Reply:
That's fair. My aesthetic is coordinated balance, but I understand your point here